data/data/manifests/bootkube/aro-imageregistry.yaml.template

```diff
@@ -0,0 +1,16 @@
+apiVersion: imageregistry.operator.openshift.io/v1
```

Comment:
This used to be in commit HACK: mark image registry storage unmanaged 
https://github.com/jewzaam/installer-aro/commit/d6e53ba050e710d9dc221012686b69bd83c539fe